### ISSUE TITLE
Kotlin and jOOQ example contribution

### ISSUE THREAD

**Issue by lukaseder** [JetBrains/kotlin-examples#50]
Hi there,

I've seen there's a [Kotlin / QueryDSL example](https://github.com/JetBrains/kotlin-examples/tree/master/gradle/kotlin-querydsl) in the repo. Would you be interested in a Kotlin / jOOQ example as well? I would be very happy to contribute one, along the lines of this:
https://github.com/jOOQ/jOOQ/blob/master/jOOQ-examples/jOOQ-kotlin-example/src/main/kotlin/org/jooq/example/kotlin/FunWithKotlinAndJOOQ.kt

Would also be happy to help maintaining it in the future.

Let me know what you think.

**Comment by yanex** [comment]
Of course, if you have a working example, we will be happy to have it in this repo 👍

**Comment by lukaseder** [comment]
Cool, thank you. I'll send a PR during the next days
